recipes/pymc3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pymc3" %}
+{% set version = "3.0rc2" %}
+{% set md5 = "80105941758b9eba47c8a2774d5ee905" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python >=3.4
```

Comment:
So do not add `skip: True  # [py2k]`, just remove the ` >=3.4` from the `build` and `run` sections. conda-forge only builds PY27, PY34, and PY35.